src/Ledger/PParams.lagda

```diff
@@ -38,24 +43,34 @@ record PoolThresholds : Set where
   field Q1 Q2a Q2b Q4 : ℚ
 
 record PParams : Set where
-  field -- Network group
+  field
+\end{code}
+\emph{Network group}
+\AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
+\begin{code}
         maxBlockSize      : ℕ
         maxTxSize         : ℕ
         maxHeaderSize     : ℕ
         maxValSize        : ℕ
         pv                : ProtVer -- retired, keep for now
-
-        -- Economic group
+\end{code}
+\emph{Economic group}
+\AgdaTarget{a, b, minUTxOValue, poolDeposit}
+\begin{code}
         a                 : ℕ
         b                 : ℕ
         minUTxOValue      : Coin
         poolDeposit       : Coin
-
-        -- Technical group
+\end{code}
+\emph{Technical group}
+\AgdaTarget{Emax, collateralPercent}
+\begin{code}
         Emax              : Epoch
         collateralPercent : ℕ
```

Comment:
The `:` here isn't aligned with all the others

Reply:
Hmmm... I didn't change these lines.  Also, it seems to me these all nearby colons are in column 26.  Did you want me to move them all to column 27?  I'm trying to avoid having too many no-op diff lines in my PRs. 😄

Reply:
Ah, I see in the pdf that they're not aligned even though all colons are in column 26 in the Agda code.  Okay, I'll move them all to column 27 and see if that fixes it.